prereise/gather/demanddata/nrel_efs/map_states.py

```diff
@@ -52,6 +101,27 @@ def decompose_demand_profile_by_state_to_loadzone(df, save=None):
     # Convert from local hours to UTC time
     df_lz = shift_local_time_by_loadzone_to_utc(df_lz)
 
+    # Create mapping from state abbreviations to loadzone IDs
+    abv2id = {}
+    for k, v in id2abv.items():
+        abv2id.setdefault(v, []).append(k)
+
+    # Determine the loadzones to be inlcuded in the profile, as specified by regions
+    loadzones = set()
+    for region in regions:
+        if len(region) == 2:
+            loadzones = loadzones | set(abv2id[region])
+        else:
+            loadzones = loadzones | interconnect2id[region]
+    loadzones = list(sorted(loadzones))
```

Comment:
I think `sorted` will return a list by default, so maybe we don't need the `list` conversion here.

Reply:
Good catch, thanks!

Reply:
Done.